src/pip/_internal/utils/misc.py

```diff
@@ -993,7 +996,11 @@ def call_subprocess(
 
 def write_output(msg, *args):
     # type: (str, str) -> None
-    logger.info(msg, *args)
+    command_logger.info(msg, *args)
```

Comment:
I would add a comment above this line saying you're logging so the output can appear in the log file (if enabled).

Also, it might be nice to distinguish these lines in the log file somehow, like by prefixing the message with `"output: "`.